src/Cache/Cache.php

```diff
@@ -168,7 +168,18 @@ protected static function _buildEngine($name)
         }
 
         $config = static::$_config[$name];
-        $registry->load($name, $config);
+
+        try {
+            $registry->load($name, $config);
+        } catch (RuntimeException $e) {
+            if (!array_key_exists('fallback', $config)) {
+                throw $e;
+            }
+            $fallbackEngine = static::engine($config['fallback']);
```

Comment:
The groups from the original engine should probably be applied to the fallback as well. You might want to use the fallback configuration as a 'template' for the actual fallback engine and use it to replace the missing/broken engine.